src/Styles.re

```diff
@@ -17,6 +17,7 @@ let app =
   ]);
 
 let textOnlyModalText = style([padding(em(2.))]);
+let totalRaisedText = size => style([fontSize(em(size))]);
```

Comment:
Lets you set the size of any text

Reply:
Cool. I still think we need the figure to be slightly smaller, and the text saying raised for conservation to be slightly bigger, just for a bit of better balance and to emphasis the amount is for conservation.